node/network/src/protocol.rs

```diff
@@ -362,7 +363,7 @@ mod tests {
     fn test_serialization() {
         let tx = types::SignedTransaction::empty();
         let message: Message<MockBlock, MockBlockHeader> =
-            Message::new(MessageBody::Transaction(tx));
+            Message::new(MessageBody::Transaction(Box::new(tx)));
```

Comment:
Why do we box it? To reduce the size of `MessageBody`?

Reply:
yep.. now that we have a real sig on SignedTransaction, rather than u128.. it is a larger variant it is significantly larger other variants. i mention this in a comment on the enum definition. let me know if it needs further clarification.